src/uct/base/uct_md.c

```diff
@@ -333,11 +333,7 @@ ucs_status_t uct_config_modify(void *config, const char *name, const char *value
 
 ucs_status_t uct_md_mkey_pack(uct_md_h md, uct_mem_h memh, void *rkey_buffer)
 {
-#if ENABLE_DEBUG_DATA
```

Comment:
can we avoid packing rkey in release mode?

Reply:
I moved the check to `uct_md_fill_md_name`, because it is used by both: normal packing function and hw tm one.